Fakebook.Posts/Fakebook.Posts.RestApi/DTOs/EditCommentDTO.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using Fakebook.Posts.Domain.Constants;
+
+namespace Fakebook.Posts.RestApi.DTOs
+{
+    public class EditCommentDTO
```

Comment:
naming convention, prefer `Dto` to `DTO` for these classes, including the namespace & folder name